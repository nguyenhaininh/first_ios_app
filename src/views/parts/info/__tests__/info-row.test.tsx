/*---------------------------------------------------------------------------------------------
 *  Copyright (c) Lunascape Corporation. All rights reserved.
 *--------------------------------------------------------------------------------------------*/
import * as React from 'react';
import InfoRow from '../info-row';

import * as renderer from 'react-test-renderer';

it('renders correctly', () => {
  const onPress = jest.fn();
  const tree = renderer.create(
    <InfoRow
      infoId="abc"
      contentText="Content Text"
      headerText="Header Text"
      onPress={onPress}
    />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
