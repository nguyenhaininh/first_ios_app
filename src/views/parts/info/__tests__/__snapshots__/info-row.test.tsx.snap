// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={null}
>
  <View
    style={
      Object {
        "alignItems": "stretch",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "height": 25,
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "height": 25,
              "justifyContent": "center",
              "width": 70,
            },
            Object {
              "backgroundColor": "#004098",
            },
          ]
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#ffffff",
              "fontSize": 11,
              "fontWeight": "bold",
            }
          }
        >
          [missing "en.information" translation]
        </Text>
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#717b91",
            "fontSize": 11,
            "marginLeft": 10,
            "marginRight": 38,
          }
        }
      >
        Header Text
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#333333",
            "flex": 1,
            "fontSize": 14,
            "fontWeight": "bold",
            "lineHeight": 21,
            "marginBottom": 27,
            "marginLeft": 16,
            "marginTop": 14,
          }
        }
      >
        Content Text
      </Text>
      <Image
        source={
          Object {
            "testUri": "../../../src/assets/images/arrow-ico.png",
          }
        }
        style={
          Object {
            "marginBottom": 27,
            "marginLeft": 10,
            "marginRight": 14,
            "marginTop": 14,
          }
        }
      />
    </View>
  </View>
</View>
`;
